packages/paste-website/src/pages/getting-started/design.mdx

```diff
@@ -75,78 +75,72 @@ If you’re unfamiliar with Abstract, version control, or git workflows, read th
+To create a branch, click *Master* in the left sidebar and then click *New Branch…* in the upper right.
 
 <Callout variant="warning">
   <CalloutTitle as="h4">Early Access! Early Access! Early Access!</CalloutTitle>  
   <CalloutText>This section will be updated in the future with specific guidance for Flex &amp; SendGrid.</CalloutText>
 </Callout>
 
-When you’re working in the Console component library, the branch naming convention is:
+When you’re working in the Console components library, the branch naming convention is:
 
 ```
-Product Designer Initials - [JIRA ticket] [description]
+[JIRA ticket] [description]
 
 For example:
 
-AI - [PROJ-1234] Added a new button
+[PROJ-1234] Added a new button
 ```
 
 2. Once you’ve created your branch, go to the Overview tab and add a link to your documentation in the Summary section and a description for context.
 
 3. To start working in Sketch, return to Files and double-click on a Sketch file.
 
-4. Once you’ve made changes, you’ll commit your work. It does not affect the Master branch. Always Be Committing!
-
-
-Any changes you make before you commit are saved locally to your computer and only viewable by you.
-
-To commit, click Commit Changes on the Abstract plugin in Sketch.
-
-When working on design system projects, please follow our commit message conventions below.
+4. Once you’ve made changes, you’ll commit your work. It does not affect the Master branch. **Always Be Committing!**
 
+**Changes in Abstract are tracked per artboard.** If you don’t want to lose track of an Abstract comment thread on an artboard, make sure you don’t delete it (even if you rename another artboard with the same name), or don’t copy-and-paste it and start working off the duplicate.
 
 <Callout>
   <CalloutTitle as="h4">What's a Commit?</CalloutTitle>
-  <CalloutText>A commit is the way you save your work so it’s publicly viewable to the rest of the team. Committing is like saving a snapshot of your work, so you can revert your work back to previous commits when you need to.</CalloutText> 
-  <CalloutText marginTop="space40">Any changes you make before you commit are saved locally to your computer and only viewable by you. To commit, click Commit Changes on the Abstract plugin in Sketch.</CalloutText>
+  <CalloutText>A commit is the way you save your work so it’s publicly viewable to the rest of the team. Any changes you make before you commit are saved locally to your computer and only viewable by you.</CalloutText> 
+  <CalloutText marginTop="space40">Committing is like saving a snapshot of your work, so you can revert your work back to previous commits when you need to. So get comfortable deleting explorations of work! If you commit before and after you delete a bunch of explorations, you’ll be able to find those explorations in the future. More importantly, your files will be easier to grok for anyone who’s looking for the latest version of the work.</CalloutText>
```

Comment:
this seems like a lot of text for a callout. Consider refactoring this to plain text.